Sources/App/Controllers/Heartbeat/HeartbeatController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  HeartbeatController.swift
+//  App
+//
+//  Created by di on 01.12.2017.
+//
+import Vapor
+import HTTP
+
+final class HeartbeatController  {
+  
+  func index(req: Request) throws -> ResponseRepresentable {
+    guard let beat = try Heartbeat.all().first else {
+      return Response.init(status: .noContent)
+    }
+    let result = try beat.makeJSON()
+    return result
+  }
+  
+  func store(req: Request) throws -> ResponseRepresentable {
+    let count = try Heartbeat.count()
+    if count > 0 {
+      try Heartbeat.makeQuery().delete()
+    }
+    let json = req.json
+    let value : Int = try json?.get("beat") ?? -1
```

Comment:
Please, refactor it a bit to`let value: Int`

Reply:
fixed